packages/vue-dot/src/elements/DialogBox/DialogBox.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+	<VDialog
+		ref="vDialog"
+		:value="dialog"
+		:width="width"
+		@input="$emit('update:dialog',$event)"
+		@keydown.esc="close"
+	>
+		<VCard
+			class="py-6 px-6"
+		>
+			<VCardTitle
+				class="pa-0 mb-5"
+			>
+				<h2 class="title font-weight-bold">
+					{{ title }}
+				</h2>
+
+				<VSpacer />
+
+				<VBtn
+					icon
+					@click="close"
+				>
+					<VIcon>
+						{{ closeIcon }}
+					</VIcon>
+				</VBtn>
+			</VCardTitle>
+			<VCardText>
+				<!-- Bloc of content  //-->
+				<VContainer>
+					<slot
+						name="content"
+						:itemProps="item"
+					/>
+				</VContainer>
+				<!-- End of bloc //-->
+			</VCardText>
+			<!-- Block of button actions //-->
+			<VCardActions class="pa-0">
+				<VSpacer />
+				<!-- button actions //-->
+				<slot
+					name="footer"
+				>
+					<VBtn
+						color="normal"
```

Comment:
```suggestion
						color="primary"
```
`normal` ce n'est pas une couleur :smile: